### PR TITLE
Docs: Test case, subtask, and overall verdicts

### PR DESCRIPTION
**Pull request by fushar**
Also changed the term "RESULT" in local grading output to "VERDICT"

### CODE REVIEW

File: docs/topic-guides/grading.rst [hunk 1]
```diff
@@ -43,11 +52,69 @@ Time Limit Exceeded
     The solution did not stop within the time limit, if specified.
 
 Internal Error
-    Custom scorer (if any) crashed or did not give valid verdict.
+    Custom :ref:`scorer <styles_scorer>` / :ref:`communicator <styles_communicator>` (if any) crashed or did not give valid verdict.
```
Comment (jonathanirvings) [inline docs/topic-guides/grading.rst]: `valid verdict` -> `a valid verdict`?

File: docs/topic-guides/grading.rst [hunk 2]
```diff
@@ -43,11 +52,69 @@ Time Limit Exceeded
...
+
+Test case verdicts
+******************
+
+The verdict of each test case will be shown. For OK statuses, the points (given by the scorer) will be also shown.
+
+Subtask verdicts
+****************
+
+If the problem has subtasks, the verdict of each subtask will be also shown. The verdict of a subtask is the combination of:
```
Comment (jonathanirvings) [inline docs/topic-guides/grading.rst]: `will be also` -> `will also be`?

alternatively, `will be shown as well`?

File: docs/topic-guides/grading.rst [hunk 3]
```diff
@@ -43,11 +52,69 @@ Time Limit Exceeded
...
+
+- status: the worst status of test case verdicts in the subtask
+- points:
+
+  - 0, if no test case verdicts in the subtask have Accepted or OK status,
```
Comment (jonathanirvings) [inline docs/topic-guides/grading.rst]: ini bukannya enggak gini yah? kalo ada at least satu testcase aja yang bukan AC/OK, even though N-1 testcase lainnya AC, dia tetep 0 kan?

Comment (fushar) [inline docs/topic-guides/grading.rst]: Bener juga, gw siwer. Fixed

**Pull request merged**